### DISCUSSION TITLE
fixing wallpaper on secondary monitor in portrait mode

### DISCUSSION THREAD

**Discussion by SketchyStunts** [prasanthrangan/hyprdots#1923] [Q&A] [answered]
Just purchased a 32:9 oled monitor & using my old 34" ultrawide in portrait mode & the wallpaper is cropped on the monitor in portrait mode. Looked at the various swww scripts in .local/share/bin but can't figure it out. A solution would be much appreciated.
![241013_12h39m25s_screenshot](https://github.com/user-attachments/assets/f8240668-0bf4-4c37-b469-4c09e8ba741d)

**Comment by kRHYME7** [comment, marked as answer]
Hi try to open this in swww github repo. Or you can try to install `swww-git` to see if the latest commits fix it for you.

There is also an open issue in Hyprland repo.

**Reply by SketchyStunts** [reply]
swww-git resolved the issue. Thanks

**Reply by kRHYME7** [reply]
https://github.com/hyprwm/Hyprland/issues/8028
